src/style.css

```diff
@@ -1 +1,118 @@
-/* Styles go here */
+@font-face {
+  font-family: Roboto;
+  font-weight: 500;
+  src: url(fonts/Roboto-Medium.ttf);
+}
+
+.html {
+  box-sizing: border-box;
+
+  font-family: Roboto, sans-serif;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.body {
+  margin: 0;
+}
+
+.link-reset {
+  color: inherit;
+  text-decoration: none;
+}
+
+.lires {
+  margin: 0;
+  padding: 0;
+
+  list-style-type: none;
+  line-height: 1;
+}
```

Comment:
No need to create separate class to reset default styles, just set this styles to elements class or apply it to tag
(you can use tags for default styles reset)